check_repo.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -e
+
+if [ $# -ne 4 ]; then
+    echo "usage: check_repo.sh <our-repo> <our-branch> <upstream-repo> <upstream-branch>"
+    exit 1
+fi
+
+OUR_REPO=$1
+OUR_BRANCH=$2
+UPSTREAM_REPO=$3
+UPSTREAM_BRANCH=$4
+
+WORK_DIR=`mktemp -d`
+clean_up() {
+    rm -rf ${WORK_DIR}
+}
+trap clean_up EXIT
+
+cd ${WORK_DIR}
+git clone ${OUR_REPO} clone
+cd clone
+
+# Git insists on knowing who we are.
+# We give it a (local to this repo) dummy config.
+git config user.email "sd-merge-check@localhost"
```

Comment:
I use `noreply@soft-dev.org` for such purposes.

Reply:
Does it bounce?

Reply:
Er, probably. I could set it up to just munch any emails. I don't think it'll make any difference in this case either way!